ruby/spec/inclusive_code/cop/inclusive_code_spec.rb

```diff
@@ -147,6 +147,28 @@ class MyInclusiveEngine
         end
       end
     end
+
+    context 'and the allowed term does not start with the non-inclusive language' do
```

Comment:
```suggestion
    context 'when the flagged term is not at the start of the allowed term' do
```
^ this reads a bit easier to me, and more closely follows [better specs](https://www.betterspecs.org/#contexts)